source/routing/tests/next_stop_time_test.cpp

```diff
@@ -2047,7 +2047,7 @@ struct classic_freq_dataset {
     classic_freq_dataset() {
         vp = type::ValidityPattern(date(2012,7,7), "11");
 
-        vj.validity_pattern = &vp;
+        vj.validity_patterns[type::RTLevel::Theoric] = &vp;
```

Comment:
Ok for not using a getter as setter :)
Would be shorter with a setter in that case (and less include).